6. Developer Guides/REST API/Channels/addAll.md

```diff
@@ -13,6 +13,7 @@ Adds all of the users of the Rocket.Chat server to the channel.
 | Argument | Example | Required | Description |
 | :--- | :--- | :--- | :--- |
 | `roomId` | `ByehQjC44FwMeiLbX` | Required | The channel's id |
+| `activeUsersOnly` | `true` | Optional <br> Default: `undefined` | Add active users only |
```

Comment:
Technically it is undefined, but for sake of understanding that the default is false what about changing this to false? Since your code to add this only checks if the value equals `true`?